client/header/activity-panel/highlight-tooltip/index.js

```diff
@@ -0,0 +1,158 @@
+/**
```

Comment:
Left it within this folder for now, although it could be moved to it's own package (might rename it in that case).
It's similar to these [WC pointers](https://github.com/woocommerce/woocommerce/blob/master/includes/admin/class-wc-admin-pointers.php#L228)